modules/users/config.ts

```diff
@@ -0,0 +1,9 @@
+export interface Config {
+    maxProfilePictureBytes: number;
```

Comment:
update this to use `UploadSize` to be consistent with uploads

Reply:
Resolved in stacked PR